server/src/test/java/io/crate/expression/predicate/FieldExistsQueryTest.java

```diff
@@ -112,6 +129,11 @@ public void test_is_not_null_matches_empty_objects() throws Exception {
         assertMatches("create table t (xs OBJECT as (x int))", false, OBJECT_VALUES);
     }
 
+    @Test
+    public void test_not_function_does_not_match_empty_objects() throws Exception {
+        assertNot("create table t (xs OBJECT)");
+    }
```

Comment:
Looks like `assertNot` doesn't do `builder.indexValues("xs", values);`.  
I think we need to test with  tricky values (already prepared in `OBJECT_VALUES`) since test name mentioned empty objects.

Reply:
Thx! I've added both version, 0 index values and the `OBJECT_VALUES` used in other tests.